requirements.txt

```diff
@@ -0,0 +1 @@
+pyyaml
```

Comment:
the `requirements.txt` is good and pretty standard for all python projects;

the dependency of graphviz you can't make a script for each distro, but document it for a few major Linux distros in README should be fine? like

```console
$ sudo apt install -y graphviz         # for Ubuntu 16+; apt-get also works
$ sudo dnf install -y graphivz         # for Fedora 24+
$ sudo pacman -S graphviz              # for ArchLinux
[...]
```